### PR TITLE
Robin/homepage hero

### PR DESCRIPTION
**Pull request by rjkerrison**
Homepage has a video hero now. It looks pretty slick.

- [ ] bleeds off the edge, centred on mobile
- [ ] on tablet, almost all the video shows
- [ ] on a particularly wide desktop, there's a border around the edge, probably the worst experience but still doesn't look terrible

### CODE REVIEW

File: src/components/homepage/hero-video.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  );
+};
+
+const HeroVideo = () => {
+  const data = useStaticQuery(graphql`
+    query {
+      file(relativePath: { eq: "homepage-hero.mp4" }) {
+        childVideoFfmpeg {
+          webm: transcode(
+            outputOptions: ["-crf 20", "-b:v 0"]
```
Comment (tkeburia) [inline src/components/homepage/hero-video.tsx:24]: might be worth adding a short comment describing what these options actually do

**Review by tkeburia — APPROVED** [review]
good stuff! 👍

File: src/components/homepage/hero-video.tsx (hunk 1 above)
Comment (rjkerrison) [reply src/components/homepage/hero-video.tsx:24]: tbh I copied and pasted an example so I'll need to look up what it's doing and check it's what we need

Comment (rjkerrison) [reply src/components/homepage/hero-video.tsx:24]: gonna merge it for now and can loop back to this after the other homepage bits

**Pull request merged**